### ISSUE TITLE
Learn How to Utilise the HPC 

### ISSUE THREAD

**Issue by grayJiaaoLi** [amosproj/amos2024ss08-cloud-native-llm#32]
## User story
1. As a developer
2. I want to learn the details about how to perform future model training on HPC
3. So that the LLM can be trained powered by HPC from FAU

## Acceptance criteria
* Get familiar with any knowledge and information about HPC
    * Be familiar with basic training procedures on HPC
    *Go through the documentation 

* Application for access permission (TU-Berlin)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by christianwielenberg** [comment]
# Working with HPC Cluster

## Accessing the cluster:
- Apply for account with either PO or johannes.jablonski@fau.de
- Login at https://portal.hpc.fau.de/ui/
- Accept invitation to use HPC at User tab
- Generate ssh key using `ssh-keygen -t ed25519 -f ~/.ssh/id_ed25519_nhr_fau`
- Upload public ssh key to hpc portal.
- edit `~/.ssh/config` to include:
```
Host csnhr.nhr.fau.de
    HostName csnhr.nhr.fau.de
    User <HPC account>
    IdentityFile ~/.ssh/id_ed25519_nhr_fau
    IdentitiesOnly yes
    PasswordAuthentication no
    PreferredAuthentications publickey
    ForwardX11 no
    ForwardX11Trusted no

Host tinyx.nhr.fau.de
    HostName tinyx.nhr.fau.de
    User <HPC account>
    ProxyJump csnhr.nhr.fau.de
    IdentityFile ~/.ssh/id_ed25519_nhr_fau
    IdentitiesOnly yes
    PasswordAuthentication no
    PreferredAuthentications publickey
    ForwardX11 no
    ForwardX11Trusted no
```

- Login with `ssh csnhr.fau.de`once, after that use `ssh tinyx.nhr.fau.de`

## Configuring Python installation
- Load python: 

To be continued
